examples/simulated_tank_gazebo/simulated_tank_gazebo.cc

```diff
@@ -0,0 +1,104 @@
+// BoB robotics includes
+#include "common/main.h"
+#include "hid/joystick.h"
+#include "robots/simulated_tank.h"
+#include "robots/gazebo_tank.h"
+#include "common/logging.h"
+#include "video/gazebocamerainput.h"
+#include "video/display.h"
+// Third-party includes
+#include "third_party/units.h"
+
+// Standard C++ includes
+#include <chrono>
+#include <iostream>
+#include <thread>
+#include <cstring>
+// Gazebo includes
+#include <gazebo/gazebo_config.h>
+#include <gazebo/transport/transport.hh>
+#include <gazebo/msgs/msgs.hh>
+#include <gazebo/common/common.hh>
+
+// Gazebo's API has changed between major releases. These changes are
+// accounted for with #if..#endif blocks in this file.
+#if GAZEBO_MAJOR_VERSION < 6
```

Comment:
Can we maybe just not support gazebo < 6?

Reply:
Good idea.